Docker/config.ini

```diff
@@ -1,15 +1,29 @@
+# Track only transactions whose scopes involve the listed accounts. Default is to track all transactions.
+# filter_on_accounts =
+
+# Limits the maximum time (in milliseconds) processing a single get_transactions call.
+get-transactions-time-limit = 3
+
 # File to read Genesis State from
-# genesis-json =
-genesis-json = "/opt/eosio/bin/data-dir/genesis.json"
+ genesis-json = /opt/eosio/bin/data-dir/genesis.json
+
+# override the initial timestamp in the Genesis State file
+# genesis-timestamp =
 
 # the location of the block log (absolute path or relative to application data dir)
 block-log-dir = "blocks"
 
 # Pairs of [BLOCK_NUM,BLOCK_ID] that should be enforced as checkpoints.
 # checkpoint =
 
-# open the database in read only mode
-readonly = 0
+# Limits the maximum time (in milliseconds) that is allowed a transaction's code to execute from a received block.
+rcvd-block-trans-execution-time = 12
```

Comment:
Where did these come from? I can't find `rcvd-block-trans-execution-time` in the code. (or am I just having a case of the mondays 🤔 )

Reply:
I had the config.ini provided to me from another developer. It looks like it may have been carried over from PR #540, which doesn't look like it made it into Dawn 3.x. I am going to remove it, and test further, and if all still works fine, I will update the PR.